### ISSUE TITLE
How to update the Badge?

### ISSUE THREAD

**Issue by SmallLars** [rrrene/inch-pages#45]
Hello René,
i added some documentation to https://github.com/SmallLars/openssl-cmac,
but i cant find out, how to update the Badge/Site. Maybe you can help me?

**Comment by rrrene** [comment]
Hi there,

Inch Pages is build with Github Pages and updated manually by myself on a semi-daily basis.

There are two pragmatic reasons for this:
1) I am currently preparing my first teaching gig next to my day job, which is at the same time exciting and time-consuming.
2) I wanted to test if people are even interested in these badges or if it all was just a pipe dream of mine. But because of reason 1 it was quiet clear that I could not invest the time to build a full-fledged CI service at this moment.

So, long story short: It is updated manually at the moment, but since it has been really well received, I plan to build an automated service in the coming weeks. When that happens, the existing badges will continue to work and update automagically.

**Edit:** I updated your badge and Inch Page: http://inch-pages.github.io/github/smalllars/openssl-cmac/

**Comment by rrrene** [comment]
Closing since we have switched to http://inch-ci.org
